pkgs/development/python-modules/libmambapy/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, cmake
+, ninja
+, libmamba
+, pybind11
+, setuptools
+, wheel
+}:
+buildPythonPackage rec {
+  pname = "libmambapy";
+  version = "1.5.7";
+  pyproject = true;
+
+  nativeBuildInputs = [ cmake ninja ];
+  buildInputs = [ libmamba pybind11 ];
+
+  src = fetchFromGitHub {
+    owner = "mamba-org";
+    repo = "mamba";
+    rev = "${pname}-${version}";
+    hash = "sha256-HfmvLi9IBWlaGAn2Ej4Bnm4b3l19jEXwNl5IUkdVxi0=";
+  };
+
+  souceRoot = "${src.name}/libmambapy";
+  cmakeFlags = [ "-GNinja" (lib.cmakeBool "BUILD_LIBMAMBAPY" true) ];
+
+  buildPhase = ''
+    runHook preBuild
+    ninjaBuildPhase
+    cp -r libmambapy ../libmambapy
+    cd ../libmambapy
```

Comment:
Do their generated ninja files not include installation rules we could reuse? Or do those not match the expectations of `pypaBuildPhase`?

Reply:
The latter:
The install phase would install them in the default `lib` output folder for cmake. The python project configuration expects the library to be in `${src.name}/libmambapy/libmambapy/*.so`